src/Bitcoin/InputSelector.h

```diff
@@ -14,7 +14,7 @@
 
 namespace TW::Bitcoin {
 
-template <typename TypeWithAmount> // TypeWithAmount has to have a uint64_t amount
+template <typename TypeWithAmount> // TypeWithAmount has to have an uint64_t amount
```

Comment:
Now this is pure lingustics, but:
'a U-Int' but
'an unsigned integer'
I would use 'a' as uint is an abbreviation.
https://oneminuteenglish.org/en/a-or-an-before-u/

Reply:
will fix, thanks

Reply:
As I said, it depends on how you read it: 'a you-int-sixtyfor' or 'an unsigned-integer-sixtyfor'. No definite correct answer